media-sound/scream/scream-3.6.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Plays sound received from network or from a QEMU Windows VM"
+HOMEPAGE="https://github.com/duncanthrax/scream"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/duncanthrax/scream.git"
+else
+	SRC_URI="https://github.com/duncanthrax/scream/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+S="${S}/Receivers/unix"
```

Comment:
dont re-reference ${S} in the definition

Reply:
Fixed